autospec/specfiles.py

```diff
@@ -581,6 +594,14 @@ def write_cmake_install(self):
             self._write_strip("popd")
             self._write_strip("rm -f %{buildroot}/usr/bin/*")
 
+        if config.config_opts['use_avx512']:
+            self._write_strip("mkdir -p %{buildroot}/usr/lib64/haswell/avx512_1")
+            self._write_strip("pushd clr-build-avx512")
+            self._write_strip("%s %s || :\n" % (self.install_macro, self.extra_make_install))
+            self._write_strip("mv %{buildroot}/usr/lib64/*so* %{buildroot}/usr/lib64/haswell/avx512_1 || :")
+            self._write_strip("popd")
```

Comment:
optional, can this be moved one line up?

Reply:
You want the popd moved up one line? Do you want that moved up in the avx2 code above this as well?

Reply:
sure, optional like I said though. little deal.

Reply:
Skipping for now.